app/preview.py

```diff
@@ -135,12 +135,16 @@ def print_letter_template():
             for line in pdf_data.read():
                 pdf_data.write(color_mapping(line))
 
-        return send_file(
+        response = send_file(
             BytesIO(pdf_data.result),
             as_attachment=True,
             attachment_filename='print.pdf'
         )
 
+        with Image(blob=pdf_data.result) as image:
```

Comment:
Can you refactor into a helper method so that the same code isn’t repeated from here: https://github.com/alphagov/notifications-template-preview/blob/8cfae14bc951c72aaf5f9b3e35762df057d7002f/app/preview.py#L49

This will mean that the same tests that cover the page count endpoint will also give us some coverage here.